### DISCUSSION TITLE
Servo Probe Precompiled Firmware

### DISCUSSION THREAD

**Discussion by Gokuh1980** [Jyers/Marlin#741] [General Discussion]
Hi, can you directly replace the BLtouch with a servo z probe? is there a possibility that a pre-compiled firmware will be added for that option? since I don't know how to use VSCODE, thank you !!!

**Comment by Cal-LF** [comment]
Good time to learn is when the need arises. Refer to #253 for steps to compile.

**Comment by smooreace** [comment]
The only person responsible for bettering you is you! It's literally a text
editor and you have to follows a few extremely simple rules.
